pkg/apis/addtoscheme_kubevirt_v1beta1.go

```diff
@@ -0,0 +1,10 @@
+package apis
```

Comment:
Wouldn't this need to be `addtoscheme_nodemaintenance_v1beta1.go`?

Reply:
fixed.